commercetools-models/src/test/java/io/sphere/sdk/shippingmethods/commands/ShippingMethodDeleteCommandIntegrationTest.java

```diff
@@ -0,0 +1,27 @@
+package io.sphere.sdk.shippingmethods.commands;
+
+import io.sphere.sdk.shippingmethods.ShippingMethod;
+import io.sphere.sdk.shippingmethods.commands.updateactions.SetKey;
+import io.sphere.sdk.shippingmethods.queries.ShippingMethodQuery;
+import io.sphere.sdk.test.IntegrationTest;
+import org.junit.Test;
+
+import static io.sphere.sdk.shippingmethods.ShippingMethodFixtures.withUpdateableShippingMethod;
+import static io.sphere.sdk.test.SphereTestUtils.randomKey;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ShippingMethodDeleteCommandIntegrationTest extends IntegrationTest {
```

Comment:
Great that we now have an integration test for the delete command :1st_place_medal: Could you please add another test for the already existing delete by ID command?